### PR TITLE
Harmonize options for parameter technology across api and bulk

### PR DESCRIPTION
**Pull request by chrwm**
#252 

Harmonizes options for parameter technology across api and bulk download.

### CODE REVIEW

**Review by FlorianK13 — APPROVED** [review]
Seems good

**Comment by chrwm** [comment]
Hi Florian, thanks for the initial review. I think the pull request is not quite ready yet for review. I just implemented the input `location`. As listed in #252, I still need and will implement in this PR the inputs:

- electricity_consumer
- balancing_area
- market
- permit
- gas
- grid

I just meant to signal that I'm working on the topic by opening the draft-pull request.

**Comment by chrwm** [comment]
I can't keep up with the current development speed, so we better merge this now or not.

File: open_mastr/utils/helpers.py [hunk 1]
```diff
@@ -249,6 +250,10 @@ def raise_warning_for_invalid_parameter_combinations(
             "(with prefix bulk_) are ignored."
         )
 
+
+def db_engine():  # TODO: Include in _create_database in MaStR() class
+    return create_engine(DB_URL)
```
Comment (FlorianK13) [inline open_mastr/utils/helpers.py]: @deniztepe If I remember correct you deleted that function, or am I wrong?

File: open_mastr/utils/helpers.py [hunk 2]
```diff
@@ -282,3 +287,73 @@ def session_scope(engine):
         raise
     finally:
         session.close()
+
+
+def technology_input_harmonisation(technology, api_data_types):
```
Comment (FlorianK13) [inline open_mastr/utils/helpers.py:288]: Maybe we could put that function to helpers.py ?

File: open_mastr/utils/helpers.py (hunk 1 above)
Comment (deniztepe) [reply open_mastr/utils/helpers.py]: Yes, it seems like that these 3 lines are slipped in during a merge.

File: open_mastr/utils/helpers.py (hunk 2 above)
Comment (deniztepe) [reply open_mastr/utils/helpers.py:288]: It is in helpers.py, isn't it?

Comment (FlorianK13) [reply open_mastr/utils/helpers.py:288]: Ah sorry, I got confused.

File: open_mastr/utils/helpers.py (hunk 1 above)
Comment (FlorianK13) [reply open_mastr/utils/helpers.py]: I deleted it again

**Comment by chrwm** [comment]
Thanks for the review! Apart from the current reviews and changes, can I go ahead and merge? @deniztepe it seems you haven't approved yet.

**Comment by deniztepe** [comment]
Seems good for me too. Sorry for delay.

**Review by deniztepe — APPROVED** [review]

**Comment by chrwm** [comment]
No worries, thank you very much!

**Pull request merged**